Resources/Prototypes/Entities/Objects/Specific/syndicate.yml

```diff
@@ -13,7 +13,6 @@
     sprite: Objects/Specific/Syndicate/telecrystal.rsi
   - type: Stack
     count: 20
-    max: 999999 # todo: add support for unlimited stacks
     stackType: Telecrystal
```

Comment:
Where you going to make Telecrystals unlimited then (in terms of maximum stack size, not in terms of inexhaustible)?

Reply:
(resolved)